### PR TITLE
proposed changes for https://github.com/AmigaPorts/AmigaCMakeCrossToolchains/issues/14

### PR DESCRIPTION
**Pull request by simpsond**
Solves #14

### CODE REVIEW

**Comment by MBeijer** [comment]
@simpsond Please resolve conflicts

**Review by MBeijer — APPROVED** [review]
Looks good, just needs to resolve the conflict first

**Comment by simpsond** [comment]
Done, Thank you!

File: m68k-amigaos.cmake [hunk 1]
```diff
@@ -74,7 +88,7 @@ endif()
 set(LIBNIX_SWAPSTACK_O ${TOOLCHAIN_PATH}/m68k-amigaos/libnix/lib/swapstack.o)
 
 # Compiler flags
-set(FLAGS_COMMON "${TOOLCHAIN_COMMON} -m${M68K_CPU} -m${M68K_FPU}-float -fomit-frame-pointer -mcrt=${M68K_CRT}")
+set(FLAGS_COMMON "${TOOLCHAIN_COMMON} -m${M68K_CPU} -m${M68K_FPU}-float -fomit-frame-pointer ${MCRT_FLAG}")
```
Comment (MBeijer) [inline m68k-amigaos.cmake]: Seems like this line was accidentally reverted when the conflict was resolved.

**Pull request merged**